tutorcruncher/hermes
# handle json signup questionnaire

Close #236 


### Technical Description

Take the tc2 `client.meta_agency.signup_questionnaire` json value from tc2, save it to the hermes Company then json dump it into a string for PD

also ignore changes to the `signup_questionnaire` in PD


### Testing

- setup as before - look at read me
- add a new custom field (check updated readme)
- signup as new company, remember to fill in to the signup questionnaire (you will need to clear application cache for signup page and flall)
- check signup questionnaire is in hermes company and pd Org
